frontend/src/components/LineChart.vue

```diff
@@ -182,17 +200,15 @@ const options = {
       // https://www.chartjs.org/docs/latest/configuration/tooltip.html
       callbacks: {
         label: function (context) {
-          // var label = context.dataset.label || '';
-          let selectedVariable = props.chosenVariable
-          let label = `${capitalizeFirstLetter(selectedVariable.abbreviation)}`
```

Comment:
Cleanup: Remove the two imports for `capitalizeFirstLetter()` and get rid of the function in _helpers/charts